cli/src/test/java/com/box/l10n/mojito/cli/command/checks/GlossaryCheckerTest.java

```diff
@@ -0,0 +1,172 @@
+package com.box.l10n.mojito.cli.command.checks;
+
+import com.box.l10n.mojito.cli.command.extraction.AssetExtractionDiff;
+import com.box.l10n.mojito.okapi.extractor.AssetExtractorTextUnit;
+import com.google.common.collect.Sets;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GlossaryCheckerTest {
+
+    private GlossaryChecker glossaryChecker;
+
+    private List<AssetExtractionDiff> assetExtractionDiffs;
+
+    @Before
+    public void setup() {
+        glossaryChecker = new GlossaryChecker();
+        glossaryChecker.setCliCheckerOptions(new CliCheckerOptions(Sets.newHashSet(), Sets.newHashSet(), "", "target/test-classes/com/box/l10n/mojito/cli/glossarychecker/glossary.json"));
+        List<AssetExtractorTextUnit> addedTUs = new ArrayList<>();
+        AssetExtractorTextUnit assetExtractorTextUnit = new AssetExtractorTextUnit();
+        assetExtractorTextUnit.setSource("A source string with Pinterest in it.");
```

Comment:
Spotted a single "Pinterest" reference remaining (you might have it fixed locally already and not pushed - just wanted to double check).